framework/src/multiapps/MultiApp.C

```diff
@@ -502,6 +502,7 @@ MultiApp::createApp(unsigned int i, Real start_time)
   // Update the MultiApp level for the app that was just created
   app->setMultiAppLevel(_app.multiAppLevel() + 1);
   app->setupOptions();
+  preRunInputFile();
```

Comment:
Who should be getting called here?  This looks out of place.  It seems like this should be calling something in the `App`...